### ISSUE TITLE
Error setting the JWTConsumer

### ISSUE THREAD

**Issue by moezubair** [ToastShaman/dropwizard-auth-jwt#22]
Hello,
I am geting the following error:

```java
The method setJwtConsumer(JwtConsumer) is undefined for the type JWTAuthFilter.Builder<SimplePrincipal>
```
when I try to:
```java
        environment.jersey().register(new AuthDynamicFeature(
                new JWTAuthFilter.Builder<SimplePrincipal>()
                    .setJwtConsumer(consumer)
                    .setRealm("realm")
                    .setPrefix("Bearer")
                    .setAuthenticator(new ExampleAuthenticator())
                    .buildAuthFilter()));
```

I am using your example, but have my own user (called simple principal)
any idea why this is happening?

**Comment by ToastShaman** [comment]
What version of dropwizard-auth-jwt are you using? 
The example you posted is for the upcoming 1.0 release of Dropwizard. 
If you are using 0.9.x look at this [example](https://github.com/ToastShaman/dropwizard-auth-jwt/blob/6c71b6a47ea1d30d658d616a185f12bedf7f175f/src/test/java/com/github/toastshaman/dropwizard/auth/jwt/example/JwtAuthApplication.java).

```
        environment.jersey().register(new AuthDynamicFeature(
                new JWTAuthFilter.Builder<MyUser>()
                        .setTokenParser(tokenParser)
                        .setTokenVerifier(tokenVerifier)
                        .setRealm("realm")
                        .setPrefix("Bearer")
                        .setAuthenticator(new ExampleAuthenticator())
                        .buildAuthFilter()));
```

**Comment by roded** [comment]
The library is currently unusable with the recently released dropwizard 1.0.0.
When can a new compatible release of dropwizard-auth-jwt be expected?

**Comment by ToastShaman** [comment]
Closing this issue. Please re-open if the issue still persists.
